docs/source/sctool/task.rst

```diff
@@ -486,27 +349,8 @@ This can be retrieved using the command ``sctool task list``.
 task stop
 =========
 
-Stops a specified task, stopping an already stopped task has no effect.
-If you want to stop all tasks see :doc:`Suspend and Resume </sctool/suspend-resume>`.
-
-**Syntax:**
-
-.. code-block:: none
-
-   sctool task stop <task type/id> --cluster <id|name> [global flags]
-
-.. include:: ../_common/task-id-note.rst
-
-task stop parameters
-.....................
-
-In addition to the :ref:`global-flags`, task stop takes the following parameter:
-
-=====
-
-.. include:: ../_common/param-cluster.rst
-
-=====
+.. datatemplate:yaml:: partials/sctool_stop.yaml
+   :template: command.tmpl
 
 Example: task stop
```

Comment:
@mmatczuk I couldn't find an equivalent command for ``task stop``. Should we delete it from the docs (rst)?